api/bazel/repository_locations.bzl

```diff
@@ -140,4 +140,26 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         license = "Apache-2.0",
         license_url = "https://github.com/bufbuild/buf/blob/v{version}/LICENSE",
     ),
+    rules_proto_grpc = dict(
```

Comment:
im not entirely clear the implications of including this here - hopefully its fine, but especially calling its toolchains it risks clashing with other things

relatedly, mobile also defines this dep

Reply:
dont worry about testing anything right now - the only thing i think we want here is to be able to access the protoc-gen-jsonschema binary - nothing else - so defo go with the absolute minimum deps at this point

i need to get up to speed again as to how the binary works with `protoc` - one thing im not clear about is even whether we need an aspect or whether it would be sufficient just to run against bootstrap, but i do rem that we need a binary to be available to protoc

lets cross those bridges once we have a ~working binary

Reply:
brilliant, thanks!